docs/api/client/reverter.rst

```diff
@@ -0,0 +1,5 @@
+:mod:`letsencrypt.client.reverter`
+---------------------------------
```

Comment:
Be careful, there are not enough `-`... You have to "underline" all characters in the headline, otherwise you'll get `WARNING: Title underline too short` (also c.f. 48f1497af6a5fd4c787013eb28a67398680aff74)